### ISSUE TITLE
Add CAIDA

### ISSUE THREAD

**Issue by gleb-svechnikov** [commondataio/dateno-public#6]
**Name this catalog**
A clear and concise catalog name
Catalog of datasets(146) from Cooperative Association for Internet Data Analysis 

**Describe the catalog**
A clear and concise description of this catalog.
Research papers and materials on internet-related technologies and protocols.

**Catalog URL**
An URL to the catalog
https://catalog.caida.org/

**More paremters**
- Catalog type: Scientific, 
- Spoken language: EN
- Software: 
- API documentation URL: https://www.caida.org/catalog/datasets/overview/

**Additional context**
Mostly pdf

**Comment by ivbeg** [comment]
Yeah, thanks. It's not indexed yet since it's relatively small. Only 146 datasets actually. And it require custom parser to add to Dateno. Still datasets are interesting, should add them some day.
